priam/src/main/java/com/netflix/priam/backup/AbstractBackup.java

```diff
@@ -165,10 +165,11 @@ public boolean isValidBackupDir(File keyspaceDir, File columnFamilyDir, File bac
             return false;        	
         }
 
-        String columnFamilyName = columnFamilyDir.getName();
+        String dirName = columnFamilyDir.getName();
 
+        String columnFamilyName = dirName.split("-")[0];
```

Comment:
Sorry, I meant to merge only the first commit. I wasn't aware, if I push to the branch again it will be added to the existing Pull Request.

I move the second patch away, because it's just work in progress.